internal/kafka/internal/services/data_plane_kafka.go

```diff
@@ -20,22 +20,37 @@ import (
 
 type kafkaStatus string
 
+func (ks kafkaStatus) String() string {
```

Comment:
I know this doesn't belong to this PR but could we rename the type `kafkaStatus` to `managedKafkaStatus` to be more specific?

Reply:
Good catch! Let's do the renaming

Reply:
done, throughout.

Reply:
:+1: